gw_eccentricity/eccDefinition.py

```diff
@@ -513,12 +513,13 @@ def process_data_dict(self,
         # Check that the dataDict contains either the hlms or the amplm/phaselm
         # and not both of these.
         for suffix in ["", "_zeroecc"]:
-            if all(key+suffix in dataDict for key in
-                   ["hlm", "amplm", "phaselm"]):
+            if "hlm"+suffix in dataDict and ("amplm"+suffix in dataDict
+                                             or "phaselm"+suffix in dataDict):
                 raise Exception(
                     f"Provide either `hlm{suffix}` or both of `amplm{suffix}` "
                     f"and `phaselm{suffix}`. `dataDict` should not contain "
-                    "all of the three at the same time.")
+                    f"`hlm{suffix}` and `amplm{suffix}` or `phaselm{suffix}` "
+                    "at the same time.")
```

Comment:
If I understand correctly, this won't raise an exception if I give only amplm in dataDict, is that correct?
Please always test code like this by making up examples.

Reply:
You probably want do an xor, which is `^` in python.
Maybe something like
if hlm is there ^ (amplm is there and philm is there):
      raise Exception
but you should test it.

Reply:
Okay, noticed the other comment. will look at the `^` thing.